lib/ansible/module_utils/facts/virtual/linux.py

```diff
@@ -68,6 +68,22 @@ def get_virtual_facts(self):
                         virtual_facts['virtualization_role'] = 'guest'
                         found_virt = True
 
+        # the lxd communication sockets are good indicators for lxc guest/host
+        # https://lxd.readthedocs.io/en/latest/dev-lxd/
```

Comment:
Looks like the documentation has moved:
```suggestion
        # https://linuxcontainers.org/lxd/docs/master/
```